### PR TITLE
VadimVildanov/HW5

### PR DESCRIPTION
**Pull request by raxtak**

### CODE REVIEW

File: VadimVildanov/HW5/Task 1.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+spisok = [1, 3, 6, 7, 10]
```
Comment (kolyasalubov) [inline VadimVildanov/HW5/Task 1.py:1]: good job

File: VadimVildanov/HW5/Task 2.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+number_fibonacci = int(input("Enter your limit: "))
```
Comment (kolyasalubov) [inline VadimVildanov/HW5/Task 2.py:1]: good job 
but look at this implementation
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')

File: VadimVildanov/HW5/Task 3.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+number = int(input("Enter your factorial: "))
```
Comment (kolyasalubov) [inline VadimVildanov/HW5/Task 3.py:1]: good job
look at this code
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)

**Review by kolyasalubov — APPROVED** [review]

**Comment by raxtak** [comment]
I have a question. Your code better than my variant? Can I fix my code when you merged?

**Pull request merged**